### DISCUSSION TITLE
Nested for

### DISCUSSION THREAD

**Discussion by Meenakshidhbw** [Meenakshidhbw/programming#9] [General]
This program uses two for loops: the outer loop controls the rows, and the inner loop prints stars (*) for each row. 

https://github.com/Meenakshidhbw/programming/pull/8

**Comment by petra-Nam** [comment]
Hey Meenakshi,  

I love how clean and effective your nested loop logic is for generating patterns! 🌟 It's always inspiring to see someone breaking down concepts like this so well. Keep up the great work—you're definitely leveling up your coding skills. Can't wait to see what creative pattern you tackle next! 🚀💻"

**Comment by Obrienmaina-Mosbach** [comment]
Hi Meenakshi,

Great work, I find your code clean and achieves its goal effectively. Keep going Meenakshi

Best,
Brian
